.iex.exs

```diff
@@ -11,3 +11,11 @@ alias Finitomata.Test.Sequenced, as: S
 require S
 alias Finitomata.Test.Transition, as: T
 require T
+
+alias Finitomata.Test.Flow.SubFlow1, as: SF
+alias Finitomata.Test.Flow, as: F
+Finitomata.start_link()
+Finitomata.start_fsm(F, "F", %{})
+Finitomata.transition("F", :to_s2)
+Finitomata.transition({:fork, :s2, "F"}, :start)
+# Finitomata.transition({:fork, :s2, "F"}, :finalize)
```

Comment:
Should this be committed?

Reply:
Depends. While I am working on this PR it should, then I’ll remove it.